src/Glpi/Form/QuestionType/AbstractQuestionType.php

```diff
@@ -75,6 +75,20 @@ public function prepareExtraData(array $input): array
         return $input; // No need to prepare the extra data
     }
 
+    #[Override]
+    public function getFormEditorJsOptions(): string
+    {
+        return <<<JS
+            {
+                "extractDefaultValue": function (question) { return null; },
+                "convertDefaultValue": function (question, old_type, value) {
```

Comment:
I do not have a solution for this limitation, but if `old_type` matches a plugin specific question, the GLPI will not be able to know what should be done.